partitioned/README.md

```diff
@@ -0,0 +1,276 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Geode partitioned region example
+
+This example demonstrates the basic property of partitioning, as well
+as what can go wrong with partitioned regions. 
+The example is presented in two parts.
+The first part shows partitioning, and the second part demonstrates
+what can go wrong with a flawed implementation.
+
```

Comment:
Actually, one more thing that I think would help with this example is if we bring up 3 servers so we can see even better how the partitioning and redundancy used the 3 available servers.